src/runtime_src/core/pcie/windows/device_windows.cpp

```diff
@@ -884,6 +884,8 @@ struct rom
       return static_cast<query::rom_ddr_bank_count_max::result_type>(hdr.DDRChannelCount);
     case key_type::rom_fpga_name:
       return std::string(reinterpret_cast<const char*>(hdr.FPGAPartName));
+    case key_type::rom_time_since_epoch:
```

Comment:
rom_time_since_epoch is needed in the xbutil validate flow, so moved this entry here to available for userpf device node as well.